examples/sunlinsol/pcg/parallel/CMakeLists.txt

```diff
@@ -14,12 +14,21 @@
 # CMakeLists.txt file for sunlinsol PCG examples
 # ---------------------------------------------------------------
 
+# Set tolerance for linear solver test based on Sundials precision
+if(SUNDIALS_PRECISION MATCHES "SINGLE")
+  set(TOL "1e-2")
+elseif(SUNDIALS_PRECISION MATCHES "DOUBLE")
+  set(TOL "1e-13")
```

Comment:
This is an awfully tight tolerance for double precision, particularly since the previous version hard-coded this to 1e-3.  Do the double and extended precision tests even pass now (without exceeding the maximum number of PCG iterations?

Reply:
Im checking, but this is what we use for some of the other iterative linear solvers.